README.md

```diff
@@ -231,6 +231,21 @@ html={(
 )}
 ```
 
+If you want to customize all tooltips, or even use your own theme, you can create your css theme as here
+https://atomiks.github.io/tippyjs/#creating-themes
```

Comment:
Current path is https://atomiks.github.io/tippyjs/#creating-custom-themes

Reply:
👍

Reply:
now it is `https://atomiks.github.io/tippyjs/themes/#creating-a-theme`, but thanks for your reminder @KubaJastrz